requests/script.js

```
@@ -19,7 +19,7 @@ const filterOptionsContainer = document.getElementById(
 );
 const applyFilterButton = document.getElementById('applyFilterButton');
 const userNameFilterInput = document.getElementById('assignee-search-input');
-let currentReqType = params.get('type') ?? OOO_REQUEST_TYPE;
+let currentReqType = params.get('type').toUpperCase() ?? OOO_REQUEST_TYPE;
 let selected__tab__class = 'selected__tab';
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'selected__tab__class' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)